iam.tf

```diff
@@ -3,62 +3,66 @@ locals {
     service_account_name = "k8s-service-account-${random_string.unique_id.result}"
     node_account_name    = "k8s-node-account-${random_string.unique_id.result}"
   }
+  create_sa = var.use_existing_sa && var.master_service_account_id != null && var.node_service_account_id != null ? true : false
```

Comment:
may be try to use variable validation on var.use_existing_sa

Reply:
Hi @romati88!
What do you mean? We cannot use another variable inside variable validate section. That's why I use locals